jquery.ajaxchimp.js

```diff
@@ -93,15 +94,31 @@ For e.g. 'http://blahblah.us1.list-manage.com/subscribe/post-json?u=5afsdhfuhdsi
                         }
                     }
 
+
                     // Translate and display message
+                    var length = 0, msgnr;
+
+                    if(Object.keys){
+                        length = Object.keys($.ajaxChimp.responses).length
+                    }
+                    else {
+                        for (var key in $.ajaxChimp.responses){
+                            if ($.ajaxChimp.responses.hasOwnProperty(key)) {
+                              length++;
+                            }
+                        }
+                    }
+
+                    while(length--) {
+                        if (msg.indexOf($.ajaxChimp.responses[length])!==-1) {
+                            msgnr = length;
+                        }
+                    }
+
                     if (
-                        settings.language !== 'en'
-                        && $.ajaxChimp.responses[msg] !== undefined
-                        && $.ajaxChimp.translations
-                        && $.ajaxChimp.translations[settings.language]
-                        && $.ajaxChimp.translations[settings.language][$.ajaxChimp.responses[msg]]
+                        settings.language !== 'en' && msgnr > -1 && $.ajaxChimp.translations && $.ajaxChimp.translations[settings.language] && $.ajaxChimp.translations[settings.language][msgnr]
                     ) {
-                        msg = $.ajaxChimp.translations[settings.language][$.ajaxChimp.responses[msg]];
+                        msg = $.ajaxChimp.translations[settings.language][msgnr];
                     }
                     label.html(msg);
```

Comment:
I would also add `msg` variable to the `settings.callback(resp, msg);` so It will be possible to make use of this messages.
BTW I look forward to merge it to master branch